CIScripts/Build.ps1

```diff
@@ -42,26 +42,16 @@ foreach ($Directory in $Directories) {
 
 try {
     Invoke-CnmPluginBuild -PluginSrcPath $Env:CNM_PLUGIN_SRC_PATH `
-        -SigntoolPath $Env:SIGNTOOL_PATH `
-        -CertPath $Env:CERT_PATH `
-        -CertPasswordFilePath $Env:CERT_PASSWORD_FILE_PATH `
-        -OutputPath $CnmPluginOutputDir `
-        -LogsPath $LogsDir
+        -OutputPath $CnmPluginOutputDir
 
     Invoke-ExtensionBuild -ThirdPartyCache $Env:THIRD_PARTY_CACHE_PATH `
-        -SigntoolPath $Env:SIGNTOOL_PATH `
-        -CertPath $Env:CERT_PATH `
-        -CertPasswordFilePath $Env:CERT_PASSWORD_FILE_PATH `
         -BuildMode $SconsBuildMode `
         -OutputPath $vRouterOutputDir `
         -LogsPath $LogsDir
```

Comment:
Are those even useful anymore? You could remove those too :D

Reply:
You mean logs?

Reply:
Yeah. I went to the logs folder and couldn't find `vrouter_build.log` for example. And even if it's there, it's rather useless, right?

Reply:
I have no idea why it is here, and I believe I would have to look through code what happens with logs, I don't feel like deleting them without it :) But I will gladly do it tomorrow :)

Reply:
I'll do it. Already working on it while writing proper test runner